packages/blockchain-wallet-v4-frontend/src/data/auth/sagas.ts

```diff
@@ -197,7 +197,7 @@ export default ({ api, coreSagas }) => {
     yield call(logout)
   }
 
-  const loginRoutineSaga = function* (mobileLogin, firstLogin, email) {
+  const loginRoutineSaga = function* (mobileLogin, firstLogin, email, restored) {
```

Comment:
instead of `mobileLogin, firstLogin, email, restored`

change to accept a config 

```
{
mobileLogin: boolean
firstLogin: boolean
email:boolean
fromRestoreFlow:boolean
}
```

loginRoutineSaga(config)